www/core/nightlies/next_patch_extension.xml

```diff
@@ -75,4 +58,23 @@
 		<maintainerurl>https://www.joomla.org</maintainerurl>
 		<targetplatform name="joomla" version="4.[123]" />
```

Comment:
```suggestion
		<targetplatform name="joomla" version="4.[1234]" />
```
Update should also be possible from a 4.4.0 to a 4.4.1-dev or later nightly, and the list XML allows that, too.

Reply:
true thanks

Reply:
So shall I apply my suggestion?